mica-core/src/main/java/org/obiba/mica/core/domain/EntityState.java

```diff
@@ -0,0 +1,70 @@
+package org.obiba.mica.core.domain;
+
+import java.io.Serializable;
+import java.util.Map;
+
+import org.springframework.data.mongodb.core.index.Indexed;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Strings;
+
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+public abstract class EntityState extends AbstractGitPersistable {
+
+  @Indexed
+  private RevisionStatus revisionStatus = RevisionStatus.DRAFT;
+
+  @Indexed
+  private String publishedTag;
```

Comment:
Could you had the publicationDate and the publishedBy properties? This is something missing in the UI (and we should not need to query git for this).